src/Fluidity/Web/UI/App_Plugins/Fluidity/propertyeditors/entitypicker.controller.js

```diff
@@ -16,6 +16,7 @@
 
         $scope.renderModel = [];
         $scope.dialogEditor = editorState && editorState.current && editorState.current.isDialogEditor;
+        $scope.isReadOnly = editorState && editorState.current && editorState.current.collectionIsReadOnly;
```

Comment:
I think i'd be tempted to pull this out into a seperate PR as I need to think whether the PE should context aware with the fluidity editor which is what this is doing.

Reply:
Yeah sure, I can take it out and add as another PR.